slick/blueprints/vm/forms.py

```diff
@@ -13,6 +13,7 @@ class CreateVMForm(Form):
     memory = SelectField('Memory (GB)', validators=[Required()])
     network = RadioField(choices=[('10', '10Mbps'), ('100', '100Mbps'),
                                   ('1000', '1000 Mbps')])
+    tags = StringField('Tags')
```

Comment:
It might be useful to add a custom validator here, as SoftLayer tags support a rather limited number of characters.
http://sldn.softlayer.com/reference/services/SoftLayer_Tag/setTags

Also, I think the default for StringField is only 20 characters long, so we might want to increase that as well.
http://wtforms.readthedocs.org/en/latest/fields.html